lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx

```diff
@@ -480,7 +478,7 @@ Graph LinearModelAnalysis::drawScaleLocation() const
   Cloud cloud(dataFull, "black", "fcircle");
   graph.add(cloud);
   // Add point identifiers for worst standardized residuals
-  UnsignedInteger identifiers(ResourceMap::GetAsUnsignedInteger("LinearModelAnalysis-Identifiers"));
+  UnsignedInteger identifiers(ResourceMap::GetAsUnsignedInteger("LinearModelAnalysis-IdentifiersNumber"));
```

Comment:
Same (here and elsewhere in this file).

Reply:
Fixed at 8efd36c846ba2aa260d71793a6c3c0e60986a8fc.